docs/Plugins/zentao.md

```diff
@@ -8,6 +8,9 @@ description: >
 
 This plugin collects Zentao data through its REST APIs. [Zentao](https://github.com/easysoft/zentaopms) is an issue-tracking tool similar to Jira.
 
+## Supported Versions
+Available for v17.x, v18.x
```

Comment:
Available for Zentao OSS v18.3, Enterprise v8.2, Flagship v4.3. Check [this doc](relative url of the supported data sources) for more details.